e2e/interchain_accounts_test.go

```diff
@@ -17,6 +17,8 @@ import (
 	"github.com/cosmos/ibc-go/e2e/testconfig"
 	"github.com/cosmos/ibc-go/e2e/testsuite"
 	"github.com/cosmos/ibc-go/e2e/testvalues"
+	feetypes "github.com/cosmos/ibc-go/v5/modules/apps/29-fee/types"
+	ibctesting "github.com/cosmos/ibc-go/v5/testing"
```

Comment:
How is this import grouping for e2e? Is this okay, or should we separate ibc-go imports?

Reply:
i dont have a strong opinion, but separation seems clearer to me